website/assets/apps/megamap/components/SearchView.jsx

```diff
@@ -292,31 +293,38 @@ export default class SearchView extends Component {
         <div className={`inner ${this.state.showFilters} ${this.state.showHelp}`}>
           <div className="searchWidget">
             <header className="searchView__header">
-              <a href="#" onClick={this.toggleFilters}>
-                <h2>FILTER</h2>
-              </a>
-              <a href="#" onClick={this.resetQueryState}>
-                RESET
-              </a>
+              <button
+                className="searchView__header-toggle"
+                onClick={this.toggleFilters}
+              >
+                <span>Close</span>
+              </button>
+              <button
+                className="searchView__header-reset-filter"
+                onClick={this.resetQueryState}
+              >
+                <span>Reset form</span>
+              </button>
             </header>
             <form onSubmit={this.handleSubmit}>
-              <div className="filterScroll">
+              <div className="filterScroll searchWidget__main">
                 <Panel
                   title="Projects"
                   ref={(el) => { this.projectsPanel = el; }}
                 >
-                  <div className="sectionRow">
-                    <label>Project Title</label>
+                  <div className="filter-input-group">
+                    <label className="filter-input-group__label">Project Title</label>
                     <input
                       type="text"
+                      className="filter-input-group__input-text"
                       value={this.state.query.name__icontains}
                       onChange={this.handleChange}
                       name="name__icontains"
                       placeholder="Project Title"
                     />
                   </div>
-                  <div className="sectionRow">
-                    <label>Status<span></span></label>
+                  <div className="filter-input-group">
+                    <label className="filter-input-group__label filter-input-group__label--with-plus">Status</label>
```

Comment:
The spec has a different field order (and slightly different labels) here, I think it should be:

1. Project Title
2. Region
3. Country
4. Project Status
5. Milestone